src/components/widgets-settings/bolt-account/bolt-account.directive.coffee

```diff
@@ -0,0 +1,66 @@
+module = angular.module('impac.components.widgets-settings.bolt-account',[])
```

Comment:
Can you review the naming please? "Bolt account" for a widget setting is not self-explanatory. Can you please reming me why we need a new setting here (versus reusing/improving the existing "account" setting?)